source/common/mongo/proxy.h

```diff
@@ -73,6 +77,31 @@ class AccessLog {
 typedef std::shared_ptr<AccessLog> AccessLogSharedPtr;
 
 /**
+ * Mongo fault configuration.
+ */
+class FaultConfig {
```

Comment:
No there is a way to write the tests without having to create a TestFaultConfig, see my comment below.